### PR TITLE
Document __scheme__ & __metrics_path__ labels available in relabeling

### PR DESCRIPTION
**Pull request by jimmidyson**

### CODE REVIEW

File: content/docs/operating/configuration.md [hunk 1]
```diff
@@ -313,7 +313,8 @@ Initially, aside from the configured global and per-target labels, a target's `j
 label is set to the `job_name` value of the respective scrape configuration.
 The `__address__` label is set to the `<host>:<port>` address of the target.
 After relabeling, the `instance` label is set to the value of `__address__` by default if
-it was not set during relabeling.
+it was not set during relabeling. The `__scheme__` & `__metrics_path__` labels
+are set to the scheme & metrics path of the target respectively.
```
Comment (juliusv) [inline content/docs/operating/configuration.md]: nit: s/&/and/

**Comment by juliusv** [comment]
:+1: otherwise, thanks!

**Comment by jimmidyson** [comment]
@juliusv Updated.

**Comment by juliusv** [comment]
Thanks!

**Pull request merged**